tests/test_data_schema.py

```diff
@@ -217,6 +217,20 @@ def testVerifyUniqueField(self):
             field = "foo"
             self.assertIsNone(SCHEMA.verify_unique_field(field))
 
+    def testExtractNameDescription(self):
+        SCHEMA.extract_name_description(SCHEMA.schema)
+        self.assertIn('nonpolymer_comp', SCHEMA.name_description_dict)
+        self.assertEqual(SCHEMA.name_description_dict.get('nonpolymer_comp'), 'Get a non-polymer chemical components described in this molecular entity.')
```

Comment:
I think it would be good to add a logging statement here for what `SCHEMA.name_description_dict.get('nonpolymer_comp')` is. Then, if the description changes and this test starts failing, it will be easy to see the description just changed, not that extract_name_description is broken.